doc/source/user_guide/style.ipynb

```diff
@@ -368,11 +366,9 @@
    "outputs": [],
    "source": [
     "def highlight_max(s):\n",
-    "    '''\n",
-    "    highlight the maximum in a Series yellow.\n",
-    "    '''\n",
-    "    is_max = s == s.max()\n",
-    "    return ['background-color: yellow' if v else '' for v in is_max]"
+    "    \"\"\"Highlight the maximum in a Series bold-orange.\"\"\"\n",
+    "    css = 'background-color: orange; font-weight: bold;'\n",
+    "    return np.where(s == np.nanmax(s.values), css, None)"
```

Comment:
you can simply use s.where no?

Reply:
i'm using the values in `s` to create a new array either with `newval1` or `newval2` conditionally. I thought with `s.where` you can either keep `existingval` or `newval1` conditionally? Very possible im missing something here..